Sources/MongoSwift/BSON/BSONDecoder.swift

```diff
@@ -54,6 +54,20 @@ public class BSONDecoder {
         case binary
     }
 
+    /// Enum representing the various strategies for decoding `Data`s from BSON.
+    ///
+    /// As per the BSON specification, the default strategy is to decode `Data`s from BSON binary types with the generic
+    /// binary subtype.
+    ///
+    /// - SeeAlso: bsonspec.org
+    public enum DataDecodingStrategy {
```

Comment:
maybe worth adding a somewhat symmetric comment in here to what you wrote in the encoding strategy about what `Data`'s default does.